### PR TITLE
Apply new design and renaming throughought sidebar and DIPs, move fil…

### PR DESCRIPTION
**Pull request by ericnakagawa**
…es around, configure site to build

Ports design changes by Dfuzr team to LIP repo. Once landed, LIP will be renamed to dip.

### CODE REVIEW

File: dips/dip-4.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
-lip: 4
+dip: 4
 title: Transaction Metadata Specification
 authors: Kevin Hurley (@kphfb)
-status: Final
+status: Last Call
 type: Informational
-created: 06/26/2020
+created: 06/26/2020d
```
Comment (oriooctopus) [inline dips/dip-4.md:7]: Seems like there's a typo here

**Review by kphfb — APPROVED** [review]

File: dips/dip-4.md [hunk 2]
```diff
@@ -22,15 +22,15 @@ ___
...
+Custodial wallets may want to identify specific users, such as merchants or individual consumers, that are the parties to an on-chain transaction.  The Diem Payment Network leverages subaddressing to provide this functionality. This document describes an approach to supporting this functionality by defining a standard that supports subaddressing to enable multiplexing a single account address into multiple wallets, enabling custodial wallets to maintain a single address rather than maintain numerous addresses.
 
 ---
 # Specification
 ---
 
 # The Lifetime of a Transaction Containing Metadata
 
-The first step to submitting a transaction is producing the metadata. The sender first produces a *Libra Canonically Serialized (LCS)* metadata_wrapper consisting of an LCS-serialized Metadata object:
+The first step to submitting a transaction is producing the metadata. The sender first produces a *Diem Canonically Serialized (LCS)* metadata_wrapper consisting of an LCS-serialized Metadata object:
```
Comment (oriooctopus) [inline dips/dip-4.md:33]: DCS?

File: dips/dip-5.md [hunk 3]
```diff
@@ -108,54 +108,54 @@ In addition to the Account Identifier standard, we propose a common serializatio
...
-  * `libra://` to explicitly specify how the identifier should be interpreted
+  * `diem://` to explicitly specify how the identifier should be interpreted
 * A base URI
   * Defines what resource the requested action is for
   * Initially, these will be account identifiers
 * Query parameters
   * Provides details for how the request should be fulfilled
   * Initially, currency code and amount to specify transaction request preferences
-    * currency uses the 'c' key, with the value encoded as a 3-letter currency code (ex. LBR)
+    * currency uses the 'c' key, with the value encoded as a 3-letter currency code (ex. XDM)
```
Comment (oriooctopus) [inline dips/dip-5.md:118]: Shouldn't this be XDX?

File: dips/dip-8.md [hunk 4]
```diff
@@ -47,7 +47,7 @@ All requests between VASPs are structured as [`CommandRequestObject`s](https://l
             "expiration_timestamp": 72322, 
             "max_cumulative_amount": {
                 "amount": 1000,
-                "currency": "LBR"
+                "currency": "XDM"
```
Comment (oriooctopus) [inline dips/dip-8.md:50]: XDX?

File: dips/dip-8.md [hunk 5]
```diff
@@ -105,8 +105,8 @@ The structure in this object can be a full pre-approval or just the fields of an
...
-| biller_address | str | Required for creation | Address of account from which billing will happen. Addresses may be single use or valid for a limited time, and therefore VASPs should not rely on them remaining stable across time or different VASP addresses. The addresses are encoded using bech32. The bech32 address encodes both the address of the VASP as well as the specific user's subaddress. They should be no longer than 80 characters. Mandatory and immutable. For Libra addresses, refer to "account identifier" section in LIP-5 for format. |
+| address | str | Required for creation | Address of account from which the pre-approval is being requested. Addresses may be single use or valid for a limited time, and therefore VASPs should not rely on them remaining stable across time or different VASP addresses. The addresses are encoded using bech32. The bech32 address encodes both the address of the VASP as well as the specific user's subaddress. They should be no longer than 80 characters. Mandatory and immutable. For Diem addresses, refer to "account identifier" section in LIP-5 for format. |
```
Comment (oriooctopus) [inline dips/dip-8.md:108]: section in LIP-5 format -> section in DIP-5 format

File: dips/dip-8.md [hunk 6]
```diff
@@ -131,13 +131,13 @@ Represents an amount and the currency type.
 
 | Field 	    | Type 	| Required? 	| Description 	|
 |-------	    |------	|-----------	|-------------	|
-| amount | uint | Y | Base units are the same as for on-chain transactions for this currency.  For example, if LibraUSD is represented on-chain where “1” equals 1e-6 dollars, then “1” equals the same amount here.  For any currency, the on-chain mapping must be used for amounts. |
-| currency | str enum | Y | One of the supported on-chain currency types - ex. LBR, etc.|
+| amount | uint | Y | Base units are the same as for on-chain transactions for this currency.  For example, if DiemUSD is represented on-chain where “1” equals 1e-6 dollars, then “1” equals the same amount here.  For any currency, the on-chain mapping must be used for amounts. |
+| currency | str enum | Y | One of the supported on-chain currency types - ex. XDM, etc.|
```
Comment (oriooctopus) [inline dips/dip-8.md:135]: xdx?

File: dips/dip-8.md [hunk 7]
```diff
@@ -131,13 +131,13 @@ Represents an amount and the currency type.
...
 
 ```
 {
     "amount": 100,
-    "currency": "LBR",
+    "currency": "XDM",
```
Comment (oriooctopus) [inline dips/dip-8.md:140]: xdx?

File: dips/dip-8.md [hunk 8]
```diff
@@ -177,24 +177,24 @@ Authorization allows the placing of holds on funds with the assurance that an am
...
 
-The [PaymentActionObject](https://lip.libra.org/lip-1/#paymentactionobject) now becomes:
+The [PaymentActionObject](https://dip.diem.com/dip-1/#paymentactionobject) now becomes:
 
 | Field 	    | Type 	| Required? 	| Description 	|
 |-------	    |------	|-----------	|-------------	|
-| amount | uint | Y | Amount of the transfer.  Base units are the same as for on-chain transactions for this currency.  For example, if LibraUSD is represented on-chain where “1” equals 1e-6 dollars, then “1” equals the same amount here.  For any currency, the on-chain mapping must be used for amounts. |
-| currency | enum | Y | One of the supported on-chain currency types - ex. LBR, etc. |
+| amount | uint | Y | Amount of the transfer.  Base units are the same as for on-chain transactions for this currency.  For example, if DiemUSD is represented on-chain where “1” equals 1e-6 dollars, then “1” equals the same amount here.  For any currency, the on-chain mapping must be used for amounts. |
+| currency | enum | Y | One of the supported on-chain currency types - ex. XDM, etc. |
```
Comment (oriooctopus) [inline dips/dip-8.md:189]: XDX

File: dips/dip-8.md [hunk 9]
```diff
@@ -177,24 +177,24 @@ Authorization allows the placing of holds on funds with the assurance that an am
...
 | action | enum | Y | Populated in the request.  This value indicates the requested action to perform. For a normal transfer, "charge" is still used.  For auth and capture, "auth" and "capture" are now available.  "capture" can only be performed after a valid "auth" |
 | valid_until | uint | N | Unix timestamp indicating the time period for which this authorization is valid.  Once this time has been reached, the authorization is no longer able to be captured and funds should be unlocked. |
 | timestamp | uint | Y | Unix timestamp indicating the time that the payment Command was created.
 
 ```
 {
     "amount": 100,
-    "currency": "LBR",
+    "currency": "XDM",
```
Comment (oriooctopus) [inline dips/dip-8.md:197]: xdx

File: docusaurus.config.js [hunk 10]
```diff
@@ -2,33 +2,34 @@ const darkCodeTheme = require("prism-react-renderer/themes/palenight");
...
   favicon: 'img/favicon.ico',
-  organizationName: 'libra',
-  projectName: 'lip',
+  organizationName: 'diem',
+  projectName: 'dip',
   themeConfig: {
     algolia: {
-      apiKey: 'cf22f07a9b6b22b6b595b439878876f8',
-      indexName: 'libra-lips',
+      apiKey: '4358e9ae3fbcd8b296ea863bd8cd9456',
```
Comment (oriooctopus) [inline docusaurus.config.js:14]: Last time I checked search was NOT working on dips

File: package.json [hunk 11]
```diff
@@ -14,7 +14,7 @@
     "@docusaurus/preset-classic": "^2.0.0-alpha.53",
     "@svgr/webpack": "^5.4.0",
     "classnames": "^2.2.6",
-    "libra-docusaurus-components": "^0.2.9",
+    "libra-docusaurus-components": "^0.2.41",
```
Comment (oriooctopus) [inline package.json:17]: should be .43

File: dips/dip-3.md [hunk 12]
```diff
@@ -1,49 +1,49 @@
...
-Conceptually, the software running on validator nodes comprises two layers. One is a blockchain core whose goal is to replicate a ledger of transactions and their outcome. The second is a framework that stores the ledger-state and a set of software modules -- encoded in the Move programming language -- that define the rules for interacting with the LPN and mutating the ledge-state (see also [LIP-2](./lip-2.md)). There are two corresponding types of upgrades to the system, (i) [updates to the core software](#software-upgrade) (ii) [updates to the framework state](#on-chain-state-upgrade).
+Conceptually, the software running on validator nodes comprises two layers. One is a blockchain core whose goal is to replicate a ledger of transactions and their outcome. The second is a framework that stores the ledger-state and a set of software modules -- encoded in the Move programming language -- that define the rules for interacting with the DPN and mutating the ledge-state (see also [LIP-2](./lip-2.md)). There are two corresponding types of upgrades to the system, (i) [updates to the core software](#software-upgrade) (ii) [updates to the framework state](#on-chain-state-upgrade).
```
Comment (oriooctopus) [inline dips/dip-3.md:20]: this link should be updated, right now it's going to lip-2

File: dips/dip-3.md [hunk 13]
```diff
@@ -1,49 +1,49 @@
...
 
 ## On-chain state upgrade
-On-chain state can only be updated via transactions. We propose three ways to effectuate changes to the LPN by directly mutating on-chain state, ordered by most frequent to least frequent: normal multi-sig transactions, multi-sig WriteSet transactions, and genesis transactions.
+On-chain state can only be updated via transactions. We propose three ways to effectuate changes to the DPN by directly mutating on-chain state, ordered by most frequent to least frequent: normal multi-sig transactions, multi-sig WriteSet transactions, and genesis transactions.
 
 ### Normal multi-sig transactions
-Normal transactions go through the VM and invoke Libra Framework modules (see [LIP-2](./lip-2.md)) to produce a Writeset that alters ledger-state. Both the resulting state change and the authenticating set of signatures are transparent and visible on-chain for auditing.
+Normal transactions go through the VM and invoke Diem Framework modules (see [LIP-2](./lip-2.md)) to produce a Writeset that alters ledger-state. Both the resulting state change and the authenticating set of signatures are transparent and visible on-chain for auditing.
```
Comment (oriooctopus) [inline dips/dip-3.md:35]: link needs to be updated

File: dips/dip-4.md [hunk 14]
```diff
@@ -183,7 +183,7 @@ For transactions under the travel rule threshold, transaction metadata inclusive
 
 For transactions over the travel rule limit, custodial to custodial transactions must exchange travel rule compliance data off-chain, so the suggested way to exchange the metadata is during this off-chain exchange. This information should not be exchanged using subaddressing.  An example of this data exchange can be seen in LIP-1.  Once the off-chain APIs have been utilized, there will be an off-chain reference ID which represents this transaction.  The on-chain transaction is now constructed.
```
Comment (oriooctopus) [inline dips/dip-4.md:184]: should be dip-1

File: dips/dip-4.md [hunk 15]
```diff
@@ -196,7 +196,7 @@ lcs_metadata = lcs.serialize(metadata, Metadata);
 
 
 // receiver_signature is passed to the sender via the off-chain APIs as per
-// https://github.com/libra/lip/blob/master/lips/lip-1.mdx#recipient-signature
+// https://github.com/diem/dip/blob/master/lips/lip-1.mdx#recipient-signature
```
Comment (oriooctopus) [inline dips/dip-4.md:199]: should be dip-1

File: dips/dip-8.md [hunk 16]
```diff
@@ -105,8 +105,8 @@ The structure in this object can be a full pre-approval or just the fields of an
...
+| address | str | Required for creation | Address of account from which the pre-approval is being requested. Addresses may be single use or valid for a limited time, and therefore VASPs should not rely on them remaining stable across time or different VASP addresses. The addresses are encoded using bech32. The bech32 address encodes both the address of the VASP as well as the specific user's subaddress. They should be no longer than 80 characters. Mandatory and immutable. For Diem addresses, refer to "account identifier" section in LIP-5 for format. |
+| biller_address | str | Required for creation | Address of account from which billing will happen. Addresses may be single use or valid for a limited time, and therefore VASPs should not rely on them remaining stable across time or different VASP addresses. The addresses are encoded using bech32. The bech32 address encodes both the address of the VASP as well as the specific user's subaddress. They should be no longer than 80 characters. Mandatory and immutable. For Diem addresses, refer to "account identifier" section in LIP-5 for format. |
```
Comment (oriooctopus) [inline dips/dip-8.md:109]: dip-5

File: dips/dip-3.md [hunk 17]
```diff
@@ -1,49 +1,49 @@
...
+Conceptually, the software running on validator nodes comprises two layers. One is a blockchain core whose goal is to replicate a ledger of transactions and their outcome. The second is a framework that stores the ledger-state and a set of software modules -- encoded in the Move programming language -- that define the rules for interacting with the DPN and mutating the ledge-state (see also [LIP-2](./lip-2.md)). There are two corresponding types of upgrades to the system, (i) [updates to the core software](#software-upgrade) (ii) [updates to the framework state](#on-chain-state-upgrade).
 
 All building blocks may be used for "business-as-usual" upgrades or to address urgent incidents.
 
 ## Software upgrade
 Specification changes go through the [LIP process](./lip-0.md) and non-specification changes (e.g. bug fixes and performance improvements) do not need to go through it. All software changes are submitted through standard Github reviews and proceed through the code review process before being merged to master.
```
Comment (oriooctopus) [inline dips/dip-3.md:25]: should be dip-0

File: dips/dip-5.md [hunk 18]
```diff
@@ -1,22 +1,22 @@
...
 
-* **intent identifier**: An intent identifier is a URI-serialized string that couples an *account identifier* with optional query parameters that specify preferences for an on-chain action. Initially, intent identifiers will be used for specifying transaction requests. In the fullness of time, we can expect using intent identifiers for sharing richer transaction metadata or identity information. an example of an intent identifier is `libra://lbr11q8mjtzdhrl6035feva9r7umfc6du7ezz300tv2hj7v5pyg?am=1000000&c=LBR`
+* **intent identifier**: An intent identifier is a URI-serialized string that couples an *account identifier* with optional query parameters that specify preferences for an on-chain action. Initially, intent identifiers will be used for specifying transaction requests. In the fullness of time, we can expect using intent identifiers for sharing richer transaction metadata or identity information. an example of an intent identifier is `diem://lbr11q8mjtzdhrl6035feva9r7umfc6du7ezz300tv2hj7v5pyg?am=1000000&c=LBR`
```
Comment (oriooctopus) [inline dips/dip-5.md:19]: lbr needs to be swapped out here for xdx in a couple of places in the final url

File: dips/dip-5.md [hunk 19]
```diff
@@ -1,22 +1,22 @@
...
+* **subaddress**: Accounts on-chain are represented by an address.  To allow multiplexing of a single address into multiple wallets, custodial wallets may use “subaddresses” under the on-chain address for each underlying user.  While custodians can keep an internal ledger for mapping subaddresses, these mapping ledgers are not stored as resources on the Diem blockchain. It is a best practice for VASPs on Diem to use subaddresses as a single-use address to remove linkability.  In this way, subaddresses serve as a many-to-one mapping between subaddresses and a user - where subaddresses are not re-used for more than one payment.
 
 * **account identifier**:  An account identifier is a base-32 encoded string that captures 1) the network version the address is intended for, 2) the address type (with or without subaddress) and 3) the underlying address components. This identifier uses the bech32 encoding which consists of a human readable prefix, delimiter, serialized payload (account address, subaddress) and checksum. An example of an account identifier is `lbr1p7ujcndcl7nudzwt8fglhx6wxn08kgs5tm6mz4usw5p72t`.
```
Comment (oriooctopus) [inline dips/dip-5.md:17]: is the lbr at the beginning of the url intentional?

File: dips/dip-5.md [hunk 20]
```diff
@@ -55,7 +55,7 @@ For communicating account identity, we propose using a compact, versioned and ca
 - Subaddresses should be single-use and should not use personal information, such as name, email address, government-issued identification number, to generate.
 
 ### Format
-The Libra Account Identifier consists of
+The Diem Account Identifier consists of
 * A prefix (also known as hrp (human readable part) which identifies the network version this address is intended for
   * “lbr” for Mainnet addresses
```
Comment (oriooctopus) [inline dips/dip-5.md:60]: should be different

File: dips/dip-5.md [hunk 21]
```diff
@@ -108,54 +108,54 @@ In addition to the Account Identifier standard, we propose a common serializatio
...
 
 ### Example of request intent
 This intent represents a request to receive funds at a given address. Since neither the amount nor currency are prefilled, the sender will define these fields.
 
 Identifier information
 * Prefix
-  * `libra://`
+  * `diem://`
 * Base URI
   * Account Identifier: `lbr1p7ujcndcl7nudzwt8fglhx6wxn08kgs5tm6mz4usw5p72t`
```
Comment (oriooctopus) [inline dips/dip-5.md:128]: lbr -> xdx (I think)

File: dips/dip-5.md [hunk 22]
```diff
@@ -108,54 +108,54 @@ In addition to the Account Identifier standard, we propose a common serializatio
...
 * Query parameters
   * none
 
-**Result**: `libra://lbr1p7ujcndcl7nudzwt8fglhx6wxn08kgs5tm6mz4usw5p72t`
```
Comment (oriooctopus) [inline dips/dip-5.md:132]: lbr -> xdx (I think)

File: dips/dip-5.md [hunk 23]
```diff
@@ -108,54 +108,54 @@ In addition to the Account Identifier standard, we propose a common serializatio
...
 This intent represents a request to receive funds in a specific currency at a given address. The amount is defined by the sender.
 
 Identifier information
 * Prefix
-  * `libra://`
+  * `diem://`
 * Base URI
   * Account Identifier: `lbr1p7ujcndcl7nudzwt8fglhx6wxn08kgs5tm6mz4usw5p72t`
 * Query parameters
   * `c=LBR` (currency preference is LBR composite token)
```
Comment (oriooctopus) [inline dips/dip-5.md:143]: lbr -> xdx (I think)

File: dips/dip-5.md [hunk 24]
```diff
@@ -108,54 +108,54 @@ In addition to the Account Identifier standard, we propose a common serializatio
...
 
 ### Example of request intent with currency specified
 This intent represents a request to receive funds in a specific currency at a given address. The amount is defined by the sender.
 
 Identifier information
 * Prefix
-  * `libra://`
+  * `diem://`
 * Base URI
   * Account Identifier: `lbr1p7ujcndcl7nudzwt8fglhx6wxn08kgs5tm6mz4usw5p72t`
```
Comment (oriooctopus) [inline dips/dip-5.md:141]: lbr -> xdx (I think)

File: dips/dip-5.md [hunk 25]
```diff
@@ -108,54 +108,54 @@ In addition to the Account Identifier standard, we propose a common serializatio
...
 * Query parameters
   * `c=LBR` (currency preference is LBR composite token)
 
-**Result**: `libra://lbr1p7ujcndcl7nudzwt8fglhx6wxn08kgs5tm6mz4usw5p72t?c=LBR`
+**Result**: `diem://lbr1p7ujcndcl7nudzwt8fglhx6wxn08kgs5tm6mz4usw5p72t?c=LBR`
```
Comment (oriooctopus) [inline dips/dip-5.md:145]: lbr -> xdx (I think)

File: dips/dip-5.md [hunk 26]
```diff
@@ -108,54 +108,54 @@ In addition to the Account Identifier standard, we propose a common serializatio
...
 
 ### Example of request intent with currency and amount specified
 This intent represents a request to receive a specific amount in a specific currency for a given address.
 
 Identifier information
 * Prefix
-  * `libra://`
+  * `diem://`
 * Base URI
   * Address: `lbr1p7ujcndcl7nudzwt8fglhx6wxn08kgs5tm6mz4usw5p72t`
```
Comment (oriooctopus) [inline dips/dip-5.md:154]: lbr -> xdx (I think)

File: dips/dip-5.md [hunk 27]
```diff
@@ -108,54 +108,54 @@ In addition to the Account Identifier standard, we propose a common serializatio
...
 * Query parameters
   * `c=LBR&am=1000000` (request for 1 LBR, currency preference is LBR composite token, amount is 1000000 µ-units)
```
Comment (oriooctopus) [inline dips/dip-5.md:156]: lbr -> xdx (I think)

File: dips/dip-5.md [hunk 28]
```diff
@@ -108,54 +108,54 @@ In addition to the Account Identifier standard, we propose a common serializatio
...
 
-**Result**: `libra://lbr1p7ujcndcl7nudzwt8fglhx6wxn08kgs5tm6mz4usw5p72t?c=LBR&am=1000000`
+**Result**: `diem://lbr1p7ujcndcl7nudzwt8fglhx6wxn08kgs5tm6mz4usw5p72t?c=LBR&am=1000000`
```
Comment (oriooctopus) [inline dips/dip-5.md:158]: lbr -> xdx (I think)

File: dips/dip-8.md [hunk 29]
```diff
@@ -105,8 +105,8 @@ The structure in this object can be a full pre-approval or just the fields of an
...
 | expiration_timestamp | uint | N | Unix timestamp indicating the time at which this pre-approval will expire - after which no funds pulls can occur.  To expire an existing pre-approval early, this field can be updated with the current unix timestamp. |
 | funds_pre_approval_id | str | Y | Unique reference ID of this pre-approval on the pre-approval initiator VASP (the VASP which originally created this pre-approval object). This value should be unique, and formatted as “<creator_vasp_onchain_address_bech32>_<unique_id>”.  For example, ”lbr1pg9q5zs2pg9q5zs2pg9q5zs2pgyqqqqqqqqqqqqqqspa3m_7b8404c986f53fe072301fe950d030de“. Note that this should be the VASP address and thus have a subaddress portion of 0. This field is mandatory on pre-approval creation and immutable after that.  Updates to an existing pre-approval must also include the previously created pre-approval ID. |
```
Comment (oriooctopus) [inline dips/dip-8.md:111]: lbr -> xdx (I think)

**Review by oriooctopus — CHANGES_REQUESTED** [review]
A lot needs to be changed in terms of naming

File: dips/dip-4.md (hunk 2 above)
Comment (ericnakagawa) [reply dips/dip-4.md:33]: it's actually changed to BCS

**Pull request merged**